src/holo-crypto/crates/holo-crypto/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "holo-crypto"
+version = "0.1.0"
+authors = ["Willem Olding <willemolding@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["lib"]
+
+[dependencies]
+base64 = "0.10.1"
+ed25519-dalek = "1.0.0-pre.2"
+wasm-bindgen = "0.2"
+
+[dependencies.argon2min]
+git = "https://github.com/Holo-Host/argon2min"
+branch = "2019-10-13-holo-config"
```

Comment:
I mentioned in mattermost as well but @yegortimoshenko what was your reasoning around forking this repo and did you make any important changes?